site/en/docs/extensions/mv3/permission_warnings/index.md

```diff
@@ -158,14 +128,14 @@ Specify the path to the extension's folder in the Extension root directory field
 {% Img src="image/BrQidfK9jaQyIHwdw91aVpkPiib2/vVw89rdJOdXFYxvgM9Sj.png",
        alt="Specify Extension Path then Click Pack Extension", height="307", width="524" %}
 
-Chrome will create two files, a `.crx` file and a `.pem` file, which contains the extension's
+Chrome will create two files, a `.crx` file and a `.pem` file, which contain the extension's
 private key.
```

Comment:
Nit: currently, the construction of this sentence could mean that the `.crx` and `.pem` files both contain the extension's private key. It would be best to rephrase this line for clarity. The most straightforward solution is to make this two independent sentences.

```suggestion
Chrome will create two files, a `.crx` file and a `.pem` file. The `.pem` file contain the private key used to sign the extension.
```